documents/invoice/bootstrapper.go

```diff
@@ -51,8 +44,14 @@ func (Bootstrapper) Bootstrap(ctx map[string]interface{}) error {
 		return fmt.Errorf("identity service not initialised")
 	}
 
+	repo, ok := ctx[documents.BootstrappedDocumentRepository].(documents.Repository)
+	if !ok {
+		return fmt.Errorf("document db repository not initialised")
+	}
+	repo.Register(&Invoice{})
```

Comment:
Is it then recommended to use `new` to reserve memory for an object? Instead of above. Just trying to get clarity for further work and set up an standard.

Reply:
I'm agnostic to this at least in this occasion as registry only uses this object to derive the type using reflection. Its basically a type representation.